### PR TITLE
Update conditional-access-admin-roles.md

### PR DESCRIPTION
**Pull request by HyperGeil**
Added missing Teams Administrator role to list

### CODE REVIEW

File: docs/includes/conditional-access-admin-roles.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-author: joflore
+author: HyperGeil
```
Comment (Court72) [inline docs/includes/conditional-access-admin-roles.md:2]:
```suggestion
author: joflore
```

File: docs/includes/conditional-access-admin-roles.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 ms.service: entra-id
 ms.subservice: conditional-access
 ms.topic: include
-ms.date: 05/02/2024
-ms.author: joflore
+ms.date: 08/27/2024
+ms.author: HyperGeil
```
Comment (Court72) [inline docs/includes/conditional-access-admin-roles.md:7]:
```suggestion
ms.author: joflore
```

**Comment by Court72** [comment]
@MicrosoftGuyJFlo

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by MicrosoftGuyJFlo** [comment]
That is not part of the core list of roles as defined in the Entra recommendation. More roles can absolutely be added to your policy as you see fit, if you are using them.

**Comment by MicrosoftGuyJFlo** [comment]
#please-close

**Pull request closed without merging**